tests/foreman/cli/test_host.py

```diff
@@ -79,3 +82,205 @@ def test_create_libvirt_without_mac(self):
             u'puppet-proxy-id': self.puppet_proxy['id'],
             u'root-pass': host.root_pass,
         })
+
+
+class HostParameterTests(CLITestCase):
+    """Tests targeting host parameters"""
+
+    @classmethod
+    def setUpClass(cls):
+        """Create host to tests parameters for"""
+        super(HostParameterTests, cls).setUpClass()
+        cls.puppet_proxy = Proxy.list()[0]
+        cls.host = entities.Host()
+        cls.host.create_missing()
+        cls.host = Host.create({
+            u'architecture-id': cls.host.architecture.id,
+            u'domain-id': cls.host.domain.id,
+            u'environment-id': cls.host.environment.id,
+            u'location-id': cls.host.location.id,  # pylint:disable=no-member
```

Comment:
@abalakh got it now. To make it more clear, can you please make a note that you are using nailgun here to construct dummy host arguments which is eventually used to make the Host using cli. This will really help others folks reading this code.